index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bulls and cows</title>
+    <link rel="stylesheet" href="./styles/main.css">
+  </head>
+  <body>
+    <h1>Bulls and cows</h1>
+    <div class="game">
+      <form class="game__form form">
+        <input
+          type="text"
```

Comment:
- better to use type number and min, max attributes
- input should have a label